### ISSUE TITLE
train_test_split

### ISSUE THREAD

**Issue by cobergmann** [cobergmann/MA_PREDICTOR#6]
- How to separate by date, industry, 
- Most recent in the test set
- Multiple deals per acquirer 
- Not time series but keep timeliness in mind
- Different test sets for team members
